api/api/services/sendgrid_mail.py

```diff
@@ -1,72 +1,90 @@
-"""module to handle sendgrid integration"""
-
 import os
-from typing import Optional, List
+import base64
 import logging
-from sendgrid import SendGridAPIClient, Mail
-from sendgrid.helpers.mail import *
+import requests
+import re
+from sendgrid import SendGridAPIClient
+from sendgrid.helpers.mail import Mail, To, Attachment, FileContent, FileName, FileType, Disposition, ContentId
+from typing import Optional
+
 
 class SendgridParameters:
     """Sendgrid parameters"""
-    DEFAULT_SENDER = os.environ.get('SENDGRID_SENDER')
-    SENDGRID_API_KEY = os.environ.get('SENDGRID_API_KEY')
+    DEFAULT_SENDER = os.getenv(
+        "SENDGRID_SENDER", "YOUR_SENGRID_SENDER")
+    SENDGRID_API_KEY = os.getenv(
+        "SENDGRID_API_KEY", "YOUR_SENGRID_KEY")
+    TEMPLATE_ID = os.getenv("SENDGRID_MFA_TEMPLATE_ID",
+                            "YOUR_SENGRID_MFA_TEMPLATE_ID")
 
 
 class SendgridClient:
     """Sendgrid integration API"""
 
-    def __init__(
-            self,
-            recipient: str,
-            sender: Optional[str] = SendgridParameters.DEFAULT_SENDER,
-        ):
-
+    def __init__(self, recipient: str, sender: Optional[str] = SendgridParameters.DEFAULT_SENDER):
         self.logger = logging.getLogger(__name__)
         logging.basicConfig(level=logging.INFO)
 
         self.sendgrid_client = SendGridAPIClient(
-                api_key=SendgridParameters.SENDGRID_API_KEY,
-        )
+            api_key=SendgridParameters.SENDGRID_API_KEY)
+        self.mail = Mail(sender, To(recipient), subject=None)
+        self.mail.template_id = SendgridParameters.TEMPLATE_ID
```

Comment:
you can remove this. in some cases, we use another template.

If you want to force the usage of a template for a specific scenario, you should write a helper function, like we do for email confirmation, [for example](https://github.com/vrn-sh/erp/blob/next/api/api/models/__init__.py#L97)